<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdditionalMnemonicAddWindow</class>
 <widget class="QDialog" name="AdditionalMnemonicAddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Additional Mnemonic Add Window</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_mainMnemonic">
     <property name="text">
      <string>Main Mnemonic</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_mainMnemonic"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_companyName">
     <property name="text">
      <string>Company Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox_companyName"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_additionalMnemonic">
     <property name="text">
      <string>Additional Mnemonic</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_additionalMnemonic"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_unit">
     <property name="text">
      <string>Unit</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboBox_unit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_conversionFormula">
     <property name="text">
      <string>Conversion Formula</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_conversionFormula"/>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="button_clean">
       <property name="text">
        <string>Clean</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_input">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
